### PR TITLE
feat: update team settings

### LINKED ISSUE #826: Team settings rework
Dependent of issue [#822](https://github.com/argos-ci/argos/issues/822)

- [x] Rename page name to « team settings »
- [x] Button « Early trial end »
- [x] Remove checkout link

Plan statuses :
  - [x] No purchase (legacy)
  - [x] Trial
    - [x] Missing payment method
    - [x] Trial canceled
    - [x] Subscription will start after trial

  - [x] Trial expired
  - [x] Pro plan
  - [x] Pro plan canceled

### PR DESCRIPTION
**Pull request by jsfez**
Fix #826

### CODE REVIEW

File: apps/graphql/src/definitions/Purchase.ts [hunk 1]
```diff
@@ -23,5 +38,50 @@ export const resolvers: IResolvers = {
     account: async (purchase, _args, ctx) => {
       return ctx.loaders.Account.load(purchase.accountId);
     },
+    plan: async (purchase) => {
+      return purchase.$relatedQuery("plan");
```
Comment (gregberge) [inline apps/graphql/src/definitions/Purchase.ts:42]: Loader better.

File: apps/graphql/src/definitions/Team.ts [hunk 2]
```diff
@@ -151,6 +154,12 @@ export const resolvers: IResolvers = {
       return team.$getInviteLink();
     },
   },
+  periodStartDate: async (account) => {
+    return account.$getCurrentConsumptionStartDate();
+  },
+  periodEndDate: async (account) => {
+    return account.$getCurrentConsumptionEndDate();
```
Comment (gregberge) [inline apps/graphql/src/definitions/Team.ts:161]: Perf problems, multi fetching etc..

Comment (gregberge) [reply apps/graphql/src/definitions/Team.ts:161]: Also it should be on `Account` type, not on `Team` and `User`.

File: apps/app/src/ui/StripePortalLink.tsx [hunk 3]
```diff
@@ -1,39 +1,43 @@
...
-
-import config from "@/config";
+import { clsx } from "clsx";
 
 import { Anchor, anchorClassNames } from "./Link";
 
+export const ContactLink = () => {
+  return (
+    <>
+      Contact Argos support via{" "}
```
Comment (gregberge) [inline apps/app/src/ui/StripePortalLink.tsx:9]: Is "via" correct in english? I think "through" is better.

File: apps/app/src/ui/StripePortalLink.tsx [hunk 4]
```diff
@@ -1,39 +1,43 @@
...
+      <Anchor href="https://discord.gg/WjzGrQGS4A" external>
+        Discord
+      </Anchor>{" "}
+      or{" "}
+      <Anchor href="mailto:contact@argos-ci.com" external>
+        by e-mail
```
Comment (gregberge) [inline apps/app/src/ui/StripePortalLink.tsx:15]:
```suggestion
        by email
```

File: apps/app/src/ui/Chip.tsx [hunk 5]
```diff
@@ -14,7 +14,7 @@ export interface ChipProps
   extends Omit<React.ComponentProps<"div">, "className"> {
   color?: ChipColor;
   icon?: React.ComponentType<any> | null;
-  scale?: "sm" | "md" | undefined;
+  scale?: "xs" | "sm" | "md" | undefined;
```
Comment (gregberge) [inline apps/app/src/ui/Chip.tsx:17]: Implementation?

**Review by gregberge — CHANGES_REQUESTED** [review]

File: apps/app/src/ui/StripePortalLink.tsx (hunk 3 above)
Comment (jsfez) [reply apps/app/src/ui/StripePortalLink.tsx:9]: While "via" is technically correct in English, if it bothers you, I can use "on" instead.

File: apps/app/src/ui/Chip.tsx (hunk 5 above)
Comment (jsfez) [reply apps/app/src/ui/Chip.tsx:17]: removed

File: apps/graphql/src/definitions/Team.ts (hunk 2 above)
Comment (gregberge) [reply apps/graphql/src/definitions/Team.ts]: I will work on it, let it like that.

**Pull request merged**